iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/twin/QueryTwinTests.java

```diff
@@ -48,7 +48,7 @@
 @RunWith(Parameterized.class)
 public class QueryTwinTests extends DeviceTwinCommon
 {
-    public static final int QUERY_TIMEOUT_MILLISECONDS = 60 * 1000; // 1 minute
+    public static final int QUERY_TIMEOUT_MILLISECONDS = 4 * 60 * 1000; // 4 minutes
```

Comment:
why are we increasing this to 4 minutes?

Reply:
Also it will be helpful to add a comment here for the same

Reply:
This test has timed out a few times recently